miner/worker_builder.go

```diff
@@ -435,6 +448,15 @@ func (w *worker) simulateBundle(
 		}
 
 		if receipt.Status == types.ReceiptStatusFailed && !containsHash(bundle.RevertingTxHashes, receipt.TxHash) {
+			// for failed tx whose state is unRevertible but itself can be dropped, we drop it and revert the state and gas pool
```

Comment:
is it possible that a failed tx can go to the logic here?
A failed tx even dont have a receipt

Reply:
fixed the comment

Reply:
seems not fixed in latest code.